server/routes/article.js

```diff
@@ -46,4 +46,11 @@ router.patch(
   articleEditValidation,
   Articles.update
 );
+
+router.get(
+  '/userArticles',
+  verifyToken,
+  getSessionFromToken,
+  Articles.userArticles
```

Comment:
Why are we not checking whether the user is verified?

Reply:
fixed thanks